index.html

```diff
@@ -5,10 +5,11 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <!-- Russ' Stylesheet -->
-    <link rel="stylesheet" href="/path/to/obnoxious.css" />
-    <link rel="stylesheet" href="main.css" />
+    <!-- <link rel="stylesheet" href="/path/to/obnoxious.css" /> -->
```

Comment:
Are these comments needed or can be deleted?

Reply:
Let's leave it for now...just for testing purposes.